api/fasit.go

```diff
@@ -247,6 +407,20 @@ func buildRequest(fasit string, alias string, resourceType string, environment s
 	return req, err
 }
 
+func buildResourcePayload(resource ExposedResource, environment, application, zone string) (string) {
+
+	payload := `{
+					"type":"` + resource.ResourceType + `",
+					"environment":"` + environment + `",
+					"application":"` + application + `",
+					"scope:{
+							"zone":"` + zone + `"
+						}"
+					}`
+
+	return payload
```

Comment:
I vote for making a struct and using json.Marshal here. A bit cleaner as you won't need a separate method with all fields as params to create the payload value.